dev/test/commands/generate_release_command_test.rb

```diff
@@ -21,6 +21,7 @@ class GenerateReleaseCommandTest < TestCase
       # Stub dependencies
       Command.any_instance.stubs(:version_file_path).returns(@version_file_path)
       Command.any_instance.stubs(:load_taxonomy)
+      Command.any_instance.stubs(:system).with("git", "tag", "v#{@version}").returns(true)
```

Comment:
Not strictly related to this change, but I realized it should have been there and didn't feel it needed its own PR 🙂